portfolio/src/main/webapp/script.js

```diff
@@ -70,3 +70,41 @@ async function deleteComments() {
   const commentContainer = document.querySelector('.comments');
   commentContainer.innerHTML = '';
 };
+
+/** Creates a map showing Columbia University and adds it to the page. */
+function createMap() {
+  const map = new google.maps.Map(document.getElementById("map"), {
+    center: { lat: 40.808037, lng: -73.961982 },
+    zoom: 16,
+  });
+
+  const schapiroMarker = new google.maps.Marker({
+    position: {lat: 40.807901, lng: -73.965172},
+    map: map,
+    title: 'Schapiro piano'
+  });
+  const schapiroInfo = new google.maps.InfoWindow({
+    content: 'This piano is an upright piano.'
+  });
+  schapiroInfo.open(map, schapiroMarker);
```

Comment:
This code is identical for the below two locations. Consider refactoring it in a function that takes lat, lng, title, content, etc, so you don't repeat yourself.